src/components/Comments.jsx

```diff
@@ -1,9 +1,37 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { useState } from "react";
+import { getArticleComments } from "../api";
+
+const Comments = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [commentsView, setCommentsView] = useState(false);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    getArticleComments(article_id).then((comments) => {
+      setComments(comments);
+      setLoading(false);
+    });
+  }, []);
 
-const Comments = () => {
   return (
     <div id="comments">
       <p>Comments List Placeholder</p>
+      <button onClick={() => setCommentsView(!commentsView)} disabled={loading}>
+        {loading ? "Loading Comments" : "Show / Hide Comments"}
+      </button>
+      {commentsView
+        ? comments.map((comment) => {
+            return (
+              <div className="comment" key={comment.comment_id}>
+                <p>{comment.body}</p>
+                <p>Votes: {comment.votes}</p>
+                <p>User: {comment.author}</p>
+                <p>Posted: {new Date(comment.created_at).toUTCString()}</p>
+              </div>
+            );
+          })
+        : null}
```

Comment:
Ideally, you should display something to the user to let them know that there are no comments rather than returning null here.